docs/events.md

```diff
@@ -0,0 +1,88 @@
+# Player Events
+A player event is a string within the player that represents an event.
+
+## Overview
+
+Player events consist of two event types:
+
+ 1. HTML5 events - various events sent by the browser when handling media embedded using the `<video>` element.
```

Comment:
HTML5 Video events
You can link to https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Media_events which holds all of them